tests/PHPStan/Type/TypeToPhpDocNodeTest.php

```diff
@@ -217,6 +218,11 @@ public function dataToPhpDocNode(): iterable
 			'literal-string',
 		];
 
+		yield [
```

Comment:
in this test-file we might need a test for `non-empty-lowercase-string`

Reply:
This is typeToPhpDoc and not PhpDocToType.

So you'll get `non-empty-string & lowercase-string`